### ISSUE TITLE
planning: verify highway construction permits - is it showing up in datastream? is it showing in visualisation?

### ISSUE THREAD

**Issue by matkoniecz** [colouring-cities/colouring-london#964]

**Comment by matkoniecz** [comment]
@polly64 Taking https://getinvolved.london.ca/victoriabridge as a test case: it appears to be not listed in data stream at all.

I searched among all descriptions for "Victoria" / "Victoria Bridge" / "through arch" and similar and have not found anything except spurious matches.

Approval process also seems distinct from building construction/modification approval.

It is also relatively standard to make such major infrastructure project using different approval method, what in general makes sense (though increases risk of baseless ignoring local residents, but permits easier to bypass complaints that should be ignored).

Should we document it somewhere?

**Comment by polly64** [comment]
yup let's doc for now in a github issue. Can we reinstate inspire stuff?

Polly Hudson
Senior Research Fellow, The Alan Turing Institute
Project Lead:  Colouring Cities Research Programme
https://www.turing.ac.uk/research/research-projects/colouring-london-and-colouring-cities-research-programme<https://eur01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.turing.ac.uk%2Fresearch%2Fresearch-projects%2Fcolouring-london-and-colouring-cities-research-programme&data=05%7C01%7Cphudson%40turing.ac.uk%7C7c9a4ff1f8bb4a7ae23d08da5ff2fb92%7C4395f4a7e4554f958a9f1fbaef6384f9%7C0%7C0%7C637927794457079591%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=jygW8H2dTvboNVAwuUMbICB50DA7hqDgy9xF3jx7AB4%3D&reserved=0>

The Alan Turing Institute is the UK's national institute for data science and artificial Intelligence.
The Alan Turing Institute is a limited liability company, registered in England with registered number 09512457 with registered offices at British Library, 96 Euston Road, London, England, NW1 2DB. We are also a charity registered in England with charity number 1162533. DISCLAIMER: Although we have taken reasonable precautions to ensure the completeness and accuracy of this e-mail, transmission cannot be guaranteed to be secure or error-free as information could be intercepted, corrupted, lost, destroyed, arrive late or be incomplete. If you receive a suspicious or unexpected email from us, or purporting to have been sent on our behalf, particularly containing different bank details, please do not reply to the email, click on any links, open any attachments, or comply with any instructions contained within it. Our Transparency Notice found here - https://www.turing.ac.uk/transparency-notice<https://eur01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.turing.ac.uk%2Ftransparency-notice&data=05%7C01%7Cphudson%40turing.ac.uk%7C7c9a4ff1f8bb4a7ae23d08da5ff2fb92%7C4395f4a7e4554f958a9f1fbaef6384f9%7C0%7C0%7C637927794457079591%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=ivPknL0rv6NzK65vESEvZqA7IXBOlWeMpSsNSAX%2BORk%3D&reserved=0> sets out how and why we collect, store, use and share your personal data and it explains your rights and how to raise concerns with us.


________________________________
